translations/it.js

```diff
@@ -6,9 +6,9 @@ var it = {
 
```

Comment:
NOTE: Italian translation file was updated recently in the original repo:
![image](https://user-images.githubusercontent.com/11869572/77394765-40830e80-6df4-11ea-8ec4-2706d8c4296c.png)

Hence all the changes

Reply:
Why does that mean that this diff contains changes to the translations? If this branch is merged with or rebased on the latest upstream changes, the translations shouldn't appear in this diff.

Reply:
Yes, I did think about merging with the original repo.. but was procrastinating 😅 I'll do that instead of just changing 1 file.

Reply:
DONE!! I fetched latest upstream changes from master, rebased the master, then merged into my branch 😁